### ISSUE TITLE
Accessibility add link name

### ISSUE THREAD

**Issue by Arhell** [dapr/website#112]
Why it matters
An accessible name is a word or phrase coded in a way that assistive technologies can associate it with a specific user interface object. Assistive technologies can then refer to the object by name, not just by type. When a link doesn’t have an accessible name, people who use assistive technologies have no way of knowing its purpose.
![Monosnap Dapr - Distributed Application Runtime - ](https://github.com/user-attachments/assets/9650c9ed-0773-4c81-b183-ea4b543b3be1)
